code/go/0chain.net/blobbercore/handler/storage_handler.go

```diff
@@ -712,6 +712,7 @@ func verifySignatureFromRequest(r *http.Request, pbK string) (bool, error) {
 	}
 
 	hash := encryption.Hash(data)
+	pbK = encryption.MiraclToHerumiPK(pbK)
```

Comment:
Feel free to merge if you want,
but i personally wouldn't do that, assigning value with changed "essense" to the same var.
maybe to call it pbKHerumi... or not creating it at all and just wrapping the passed param with encryption.MiraclToHerumiPK().